change to branch

OliveCannon opened the pull request:
spelling change to branch

OliveCannon committed:
change to branch

spelling change to branch

yuhattor commented:
@OliveCannon thanks for the PR!
Coult you take a look at the PR #484 comment?

yuhattor commented:
@OliveCannon thanks for the PR!
Coult you take a look at the PR #484 comment?

OliveCannon committed:
Adding March 9th event

Adding March 9th event - I don't know how to change the name of the branch but March 9th is correct

OliveCannon committed:
adding March 9th event

Adding March 9th event.  March 9th is correct bu I didn't know how to rename the branch

yuhattor commented:
see https://github.com/InnerSourceCommons/innersourcecommons.org/pull/491
